sdk/keyvault/Azure.Security.KeyVault.Keys/src/JsonWebKey.cs

```diff
@@ -503,6 +501,31 @@ internal void WriteProperties(Utf8JsonWriter json)
 
         void IJsonSerializable.WriteProperties(Utf8JsonWriter json) => WriteProperties(json);
 
+        private static Func<RSAParameters, RSA> s_rsaFactory;
+        private static RSA CreateRSAProvider(RSAParameters parameters)
+        {
+            if (s_rsaFactory is null)
+            {
+                MethodInfo createMethod = typeof(RSA).GetMethod(nameof(RSA.Create), BindingFlags.Public | BindingFlags.Static, null, new[] { typeof(RSAParameters) }, null);
+                if (createMethod != null)
+                {
+                    s_rsaFactory = (Func<RSAParameters, RSA>)createMethod.CreateDelegate(typeof(Func<RSAParameters, RSA>));
+                }
+                else
+                {
+                    s_rsaFactory = p =>
+                    {
+                        RSA rsa = RSA.Create();
```

Comment:
On Framework, this won't create the CNG version of RSA, which later won't support RSA-OAEP-SHA256.